src/components/projectDetail/AddressAddModal.tsx

```
@@ -0,0 +1,110 @@
+  return (
+    <Modal
+      isOpen={isOpen}
+      onClose={onClose}
+      size="md"
+      showCloseButton={false}
+      className="rounded-2xl p-6 shadow-xl"
+    >
+      <div className="mb-6 flex items-center justify-between">
+        <h4 className="text-xl font-bold">새 배송지 추가</h4>
+        <button onClick={onClose} className="rounded-full p-1 hover:bg-gray-100">
+          <X className="h-6 w-6" />
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)